Dockerfile

```diff
@@ -20,3 +20,11 @@ ENV NODE_RESTART "0"
 EXPOSE 3000
 
 CMD ["/usr/local/bin/yarn", "start"]
+
+
+FROM image AS test
+ADD test test/
+RUN yarn run test
+
+
+FROM image
```

Comment:
What is this last `FROM image` doing? Is it the last `FROM` that wins? so the final docker image will be same as `image`, e.g without the `test` stage.

Reply:
Yep. From this [article on multi-stage docker build](https://blog.alexellis.io/mutli-stage-docker-builds/):

> The general syntax involves adding FROM additional times within your Dockerfile - whichever is the last FROM statement is the final base image.

Nice 👍